### ISSUE TITLE
contenair deploying when jettison and making server crash + unable to deploy any other way

### ISSUE THREAD

**Issue by idolord** [EvEmu-Project/evemu_Crucible#199]
**Describe the bug**
server crash when jettisoning container

**To Reproduce**
1. in chat type /create 3466
2. right click on the medium secure container an select launch for self 
3. notice the server: [InvError] ShipBound::Handle_Drop() - Item Medium Secure Container (cat 2) is neither drone, structure or deployable. (yeah those are celestial not "deployable per say eve if ccp deem them so => [https://everef.net/market/404/all](url)
4. then right click on the medium secure container again an select launch for self 
5. barce for ... nothing ... server halted but i can't see why because 
> Missing separate debuginfos, use: dnf debuginfo-install boost-system-1.73.0-10.fc33.x86_64 glibc-2.32-10.fc33.x86_64 libgcc-10.3.1-1.fc33.x86_64 libstdc++-10.3.1-1.fc33.x86_64 mariadb-connector-c-3.1.13-1.fc33.x86_64 openssl-libs-1.1.1l-2.fc33.x86_64 systemd-libs-246.15-1.fc33.x86_64 tinyxml-2.6.2-21.fc33.x86_64 zlib-1.2.11-23.fc33.x86_64
6. restart server
7. container is deployed in space and info window is broken.

**Expected behavior**
container souldn't jettison they should be deployed for self or corp 

**Screenshots**

**System Details (please complete the following information):**
 - Server OS: Ubuntu 20.04 Server
 - Docker
 - EVEmu  Version 0.8.4

**request**
any help got figur out the Missing separate debuginfos stuff would be apreciated.

**Comment by jdhirst** [comment]
@idolord is this fixed in #201?

**Comment by idolord** [comment]
No, i encoutered this Bug After on my staged branch which should be on par with stage After #201

**Comment by idolord** [comment]
the error that occure is: 

> db_1      | 2022-03-26 23:35:49 11 [Warning] Aborted connection 11 to db: 'evemu' user: 'evemu' host: '192.168.64.3' (Got an error reading communication packets)
> server_1  | /src/utils/container-scripts/start.sh: line 31:    35 Segmentation fault      (core dumped) ./eve-server
> evemu_server exited with code 139
